app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,165 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light_gray: #F2F2F2;
+$white: #FFF;
+$beach-blue: #5bc0de;
+
+body {
+  padding-top: 20px;
+}
+
+#index-background-header {
+  text-align: center;
+  padding-top: 300px;
+  font-family: Circular,"Helvetica Neue",Helvetica,Arial,sans-serif;
+  color: #FFF;
+  font-size: 5em;
+  font-weight: bold;
+}
+#search-bar-container .well{
+  background: #000;
+  background: rgba(0,0,0,.1);
+  position: absolute;
+  bottom: 0;
+  margin-bottom: 0;
+  left: 0;
+  right: 0;
+  border-color: transparent;
+}
+.columns {
+  padding-top: 150px;
+}
+.index-containers {
+  margin-top: 100px;
+}
+.index-containers > * {
+  color: #FFF;
+}
+.index-containers > h1 {
+  color: inherit;
+}
+.index-containers > p {
+  color: inherit;
+}
+
+#beach-snapshot {
+  span {
+    color: darken($light_gray, 30%);
+    padding-bottom: 100px;
+  }
+  h1 {
+    margin-bottom: 3px;
+  }
+  margin-bottom: 15px;
+}
+
+#beach-snapshot-icon-container {
+  margin-top: 25px;
+}
+
+.center {
+  text-align: center;
+}
+
+.inline-block {
+  display: inline-block;
+}
+
+.amenity-name {
+  vertical-align: 9px;
+}
+
+.col-centered {
+    display:inline-block;
+    float:none;
+    text-align:center;
+}
+
+.section-title {
+  display: inline-block;
+  vertical-align: 20px;
+}
+
+footer {
+  background: $beach-blue;
+  a, span {
+    color: $white;
+    opacity: .8;
+  }
+  a:hover {
+    color: $white;
+    text-decoration: none;
+  }
+  h3 {
+    color: $white;
+  }
+}
+
+hr {
+  border-color: lighten(gray, 30%);
+}
+
+#popular-beach-container a, .beach-container a {
+  color: $white;
+  &:hover {
+    opacity: .9;
+    text-decoration: none;
+  }
+}
+
+.beach-container {
+  margin-bottom: 50px;
+}
+
+.beach-index-location-header {
+  color: white;
+  opacity: .7;
```

Comment:
`.7` should be written with a leading zero as `0.7`<br>Properties should be ordered color, font-size, opacity